mosad_backend/src/main/resources/application.properties

```diff
@@ -1,7 +1,14 @@
-spring.application.name=mosad_backend
-server.port = ${BACKEND_SERVER_PORT:8080}
+spring.datasource.url=jdbc:postgresql://localhost:5432/mosad
```

Comment:
There is missing code segments and some merging conflicts command. Check those and remove those